### ISSUE TITLE
Support for hg38 from fastq to ranked variants

### ISSUE THREAD

**Issue by henrikstranneheim** [Clinical-Genomics/MIP#95]
**Recipes**
- [x] fastqc
- [x] bwa_mem
- [x] picardtools_mergesamfiles
- [x] markduplicates
- [x] gatk_baserecalibration
- [x] chanjo_sexcheck
- [x] samtools_subsample_mt
- [x] tiddit_coverage
- [ ] sambamba_depth (test new exons file: "grch38_scout_exons-2019-09-16-.bed";  Clinical-Genomics/scout#644)
- [x] picardtools_collectmultiplemetrics
- [ ] picardtools_collecthsmetrics (test new exons file: "grch38_scout_exons-2019-09-16-.bed";  Clinical-Genomics/scout#644)
- [x] cnvnator_ar
- [x] expansionhunter 3.0.0 with 3.0.1 var_cat_file 
- [x] manta
- [x] tiddit
- [x] delly_call
- [x] delly_reformat
- [x] sv_combinevariantcallsets
- [ ] sv_annotate (works with INFO field reduced gnomad file reset memory to 7Gb)
  Needs liftover files to grch38: 
  - grch37_svdb_query_clingen_cgh_benign_-v1.0.0-.vcf (Daniel: ?)
  - grch37_svdb_query_clingen_cgh_pathogenic_-v1.0.0-.vcf (Daniel: ?)
  - grch37_svdb_query_clingen_ngi_-v1.0.0-.vcf (Daniel: Skip)
  - grch37_mip_sv_svdb_export_-2018-10-09-.vcf (Rerun cases ~200)
  - grch37_swegen_concat_sort_-20170830-.vcf (Waiting: Jesper)
  - grch37_svdb_query_decipher_-v1.0.0-.vcf (Daniel investigates)
  - grch37_gnomad.genomes_-r2.1.1_sv-.vcf (Daniel: Skip)
- [x] sv_varianteffectpredictor
- [x] sv_vcfparser
- [ ] sv_rankvariant
  Need update to grch38:
   - grch37_cust003-cmms-red-pen_-2017-.tsv (Clinical-Genomics/scout#1300)
- [x] sv_reformat
- [ ] vcf2cytosure_ar (dnil comment below)
- [x] gatk_haplotypecaller (but long runtime 21 h,solved with reduced gnomad file)
- [x] gatk_genotypegvcfs
  Need to reprocess for grch38 (WES)
   - gatk_genotypegvcfs_ref_gvcf: grch37_gatk_merged_reference_samples.txt (Rerun cases ~30-40 WES)
- [x] gatk_gathervcfs
- [x] gatk_variantrecalibration (Works with maxgaussian 6 (case/data dependent; (livingox))
- [x] gatk_combinevariantcallsets
- [x] rtg_vcfeval
- [x] peddy
- [x] plink_ar
- [x] variant_integrity_ar
- [x] gatk_variantevalall
- [x] prepareforvariantannotationblock
- [x] rhocall
- [x] frequency_filter
- [x] cadd_ar
- [x] varianteffectpredictor
- [x] vcfparser_ar
- [ ] snpeff (Moved from snpSift to VEP plugin for grch38)
  Need update to grch38:
   - grch37_loqusdb_snv_indel_-2018-12-18-.vcf.gz (Rerun cases and moonso/loqusdb#65)
- [ ] rankvariant
  Need update of reduced_penetrance file
   - grch37_cust003-cmms-red-pen_-2017-.tsv (Clinical-Genomics/scout#1300)
- [x] endvariantannotationblock
- [x] gatk_variantevalexome
- [x] qccollect_ar
- [x] multiqc_ar
- [x] analysisrunstatus
- [x] sacct

**Comment by henrikstranneheim** [comment]
Done as far as we can go internally with MIP. Will open new issues for dealing with what is left in terms of annotation, building data references and updating rank model.

**Comment by dnil** [comment]
Native gnomAD hg38 available with their v3! See eg https://twitter.com/LFranciol/status/1184515095029800962

**Comment by dnil** [comment]
Hm, I was doing some maintenance on vcf2cytosure and realise it should not be marked hg38 ready as of yet. It a) needs a reference CGH def for hg38 and b) has some hard-coded contig sizes etc that need generalising.

**Comment by henrikstranneheim** [comment]
Noted! Let me kow when you are ready.

**Comment by jemten** [comment]
@dnil, any updates on that vcf2cytosure file?

**Comment by dnil** [comment]
> @dnil, any updates on that vcf2cytosure file? 

Right, no, but I'll pass the ping on and hopefully be back soon with news!
